sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/EventHubEndToEndTests.cs

```diff
@@ -31,21 +31,24 @@ public class EventHubEndToEndTests : WebJobsEventHubTestBase
     {
         private static readonly TimeSpan NoEventReadTimeout = TimeSpan.FromSeconds(5);
 
-        private static EventWaitHandle _eventWait;
+        private static EventWaitHandle _eventWait1;
```

Comment:
Any way that we can give them names that associate them with a context - or do we think the context can change and 1,2 give us flexibility we need to avoid confusion?

Reply:
Yeah it is pretty much just for tests that require two wait handles - no specific context needed.